### DISCUSSION TITLE
A problem occurred after creating a file and starting it.

### DISCUSSION THREAD

**Discussion by ResenPro** [thuliteio/doks#992] [Q&A] [answered]
I have two categories of folders, respectively docs minecraft.
However, when I create a new file in the minecraft folder, creating _index.md and changing draft to false starts it without any problems.
But when I create another file with another name, for example abc.md and change the draft to false, that problem happens after startup.
This problem does not occur in the docs folder.

Error: Error building site: failed to render pages: render of "section" failed: "E:\WorkSpaces\Hugo\docs\layouts\_default\list.html:19:13": execute of template failed: template: _default/list.html:19:13: executing "main" at <partial "main/blog-meta.html" .>: error calling partial: "E:\WorkSpaces\Hugo\docs\layouts\partials\main\blog-meta.html:1:17": execute of template failed: template: partials/main/blog-meta.html:1:17: executing "partials/main/blog-meta.html" at <len .Params.contributors>: error calling len: reflect: call of reflect.Value.Type on zero Value

**Comment by h-enk** [comment, marked as answer]
Did you copy the [`./layouts/docs/`](https://github.com/h-enk/doks/tree/master/layouts/docs) folder and paste it as [`./layouts/minecraft/`](https://github.com/h-enk/doks/tree/master/layouts/docs)?

And, did you create your  `minecraft` folder like so

```bash
npm run create -- --kind docs minecraft
```

So, that it mimic this folder: https://github.com/h-enk/doks/tree/master/archetypes/docs

**Reply by ResenPro** [reply]
Thanks, I have solved this problem.

**Comment by rishab002** [comment]
Is it possible to create similar for blog archetypes ?
I tried : npm run create -- --kind blog <Name>

Gives an error. I'm not sure how to implement this ?
